app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email_try_later.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.helptosavefrontend.models.HtsContext
+
+@()(implicit htsContext: HtsContext, request: Request[_], messages: Messages)
+
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(
+title = messages("hts.cannot-change-email-try-later.page-title"),
+sidebarLinks = None,
+bodyClasses = None) {
+
+ <h1 class="heading-xlarge">@Messages("hts.cannot-change-email-try-later.title")</h1>
+
```

Comment:
is this missing copy?

Reply:
how do you mean?

Reply:
I meant is this file missing the content  that was supposed to go in it - turns out it is but the addition of the actual content is part of another ticket